python/wpt/exporter/github.py

```diff
@@ -72,7 +72,9 @@ def get_branch(self, name: str) -> GithubBranch:
         return GithubBranch(self, name)
 
     def get_open_pull_request_for_branch(
-        self, branch: GithubBranch
+        self,
+        github_username: str,
```

Comment:
I don't get it, this parameter isn't used anywhere?

Reply:
This was lost after a bit of debugging, but I've restored the usage now.